subcommand/inject-connect/command.go

```diff
@@ -501,34 +501,6 @@ func (c *Command) Run(args []string) int {
 		}()
 	}
 
-	if c.flagEnableHealthChecks {
-		healthResource := connectinject.HealthCheckResource{
-			Log:                 logger.Named("healthCheckResource"),
-			KubernetesClientset: c.clientset,
-			ConsulScheme:        consulURL.Scheme,
-			ConsulPort:          consulURL.Port(),
-			Ctx:                 ctx,
-			ReconcilePeriod:     c.flagHealthChecksReconcilePeriod,
```

Comment:
Hrm. Sorry I did remove all of these, I wonder how that commit got lost :(

Reply:
OK I sorted it out. I decided in my head that I'd remove the flags in the PR that removes the controller because I could rip out all the signal handling and flags all at once. Then I came back to this one and half-did it here.
I'm going to remove the health checks flags but leave the controller ones here in this PR.